common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -494,4 +494,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Midterms interactive header switch
+  val InteractiveHeaderSwitch = Switch(
```

Comment:
I find double negatives confusing to reason about. Could we have the safe state by `On` and then [here](https://github.com/guardian/frontend/pull/20619/files#diff-cddbf1a20ebe144ec9aa35d41a46dbb4R885) we can take off the `!` and just have `InteractiveHeaderSwitch.isSwitchedOn` instead.